src/integration-test/data/test-data.ts

```diff
@@ -107,7 +107,7 @@ export function createClaimant (type: PartyType): Party {
       line1: '234 Acacia Road',
       line2: 'a really cool place',
       city: 'Edinburgh',
-      postcode: 'EDE 1AC'
```

Comment:
any reason for changing this? I've tried it on your preview app and it seems to work fine

Reply:
This is an invalid postcode format.

Reply:
Seems to pass your current validation?

Reply:
`EDE 1AC` was failing validation as invalid format, so have to update this.